agent/structs/config_entry_test.go

```diff
@@ -427,6 +427,48 @@ func TestDecodeConfigEntry(t *testing.T) {
 				},
 			},
 		},
+		{
```

Comment:
Kitchen sink config entry tests exist in 3 places. You're only missing `command/config/write/config_write_test.go`